tests/unit/util/hse_err_test.c

```diff
@@ -57,24 +60,24 @@ MTF_DEFINE_UTEST(hse_err_test, merr_test_1)
 
     (void)merr_strinfo(err, errinfo, sizeof(errinfo), 0);
     errmsg = strerror_r(merr_errno(err), errbuf, sizeof(errbuf));
-    ASSERT_EQ(0, strcmp(strstr(errinfo, "I"), errmsg));
+    ASSERT_EQ(0, strncmp(strstr(errinfo, "I"), errmsg, strlen(errmsg)));
```

Comment:
Prob need an ASSERT_NE(NULL, strstr(errnor, "I", errmsg)) before passing the output of strstr() to strcmp().  Not clear to me why you changed it to strncmp()...

Reply:
this is needed now, because the strerror_r returns the string embedded(in the middle) of our new format, not appended as it was before.